llvm/utils/lit/tests/unit/TestRunner.py

```diff
@@ -2,6 +2,7 @@
 #
 # END.
 
+# REQUIRES: python3
```

Comment:
I think this `REQUIRES:` is being ignored because it comes after `END.`.  I'm also not sure this is the right fix, since this test passes locally for me with python2.7.